package.json

```diff
@@ -187,7 +188,9 @@
     "start:mock": "MOCK=true npm run start:proxy",
     "start:mock:beta": "MOCK=true npm run start:proxy:beta",
     "prepare": "husky install",
-    "commitlint": "commitlint --from HEAD~1 --to HEAD --verbose"
+    "commitlint": "commitlint --from HEAD~1 --to HEAD --verbose",
+    "postinstall": "ts-patch install",
+    "patch-hosts": "fec patch-etc-hosts"
```

Comment:
I don't think we need to add this. If we would we would want to add it to all apps, and we probably don't want that. 

If someone wants to run the patch hosts script they can do it via `npx fec patch-etc-hosts` as well.